.travis.yml

```diff
@@ -12,6 +12,9 @@ before_install:
 - gem update --system
 - gem --version
 - bundle --version
+env:
+  global:
+    - CHEF_LICENSE=accept-no-persist
```

Comment:
Probably needs to go into appveyor as well.

Reply:
Thanks, incoming